frontend/app/dashboard/src/views/dashboard/settings/modules/members/ImportMembersQuestionsView.vue

```diff
@@ -643,119 +630,119 @@ export default class ImportMembersQuestionsView extends Mixins(NavigationMixin)
-                if (member.synced) {
-                    // Already synced: prevent doing it twice
-                    continue
-                }
-                const family =  new FamilyManager(this.$memberManager, [])
+    try {
+        // Add all members that do not yet exist
+        autoAssignMembers($members.value)
 
-                if (member.equal) {
-                    // Merge data (this is an edge case)
-                    member.equal.details!.merge(member.details)
-                    await family.patchAllMembersWith(member.equal)
+        // TODO: group family
+        for (const member of $members.value) {
+            if (member.synced) {
+                // Already synced: prevent doing it twice
+                continue
+            }
+            const family = new FamilyManager($memberManager, [])
 
-                    const patchRegistrations: PatchableArrayAutoEncoder<Registration> = new PatchableArray()
-                    const registration = this.buildRegistration(member)
+            if (member.equal) {
+                // Merge data (this is an edge case)
+                member.equal.details!.merge(member.details)
+                await family.patchAllMembersWith(member.equal)
 
-                    if (registration !== null) {
-                        // Okay to add: no duplicate
-                        patchRegistrations.addPut(registration)
-                        await family.patchMemberRegistrations(member.equal, patchRegistrations)
+                const patchRegistrations: PatchableArrayAutoEncoder<Registration> = new PatchableArray()
+                const registration = buildRegistration(member)
 
-                        // Delete conflicting registrations (based on categories too!)
-                        const deleteRegs = this.getOverrideRegistrations(registration, member)
-                        for (const r of deleteRegs) {
-                            patchRegistrations.addDelete(r.id)
-                        }
+                if (registration !== null) {
+                    // Okay to add: no duplicate
+                    patchRegistrations.addPut(registration)
+                    await family.patchMemberRegistrations(member.equal, patchRegistrations)
+
+                    // Delete conflicting registrations (based on categories too!)
+                    const deleteRegs = getOverrideRegistrations(registration, member)
+                    for (const r of deleteRegs) {
+                        patchRegistrations.addDelete(r.id)
                     }
-                } else {
-                    const registration = this.buildRegistration(member)
-                    await family.addMember(member.details, registration ? [registration] : [])
                 }
-                member.synced = true
+            } else {
+                const registration = buildRegistration(member)
+                await family.addMember(member.details, registration ? [registration] : [])
```

Comment:
Heel dit stuk is eigenlijk logica dat herschreven moet worden.

Vroeger vervingen we gewoon de volledige member.details bij elke wijziging. Dat mogen we nu niet langer doen. We moeten de aanpassingen doorgeven als een patch zoals overal in het systeem, op die manier verhinderen we collissions, maar ook aanpassingen aan gegevens die niet aangepast mogen worden (geen permissies etc - het kan dat je maar een deel van de details krijgt, dus je mag die niet zo naar de server terugsturen - die zal het ook niet meer accepteren trouwens).

Daarom is er nu de nieuwe `PlatformMember` class. Dit is allemaal super nieuw, dus ik begrijp dat je dit hier nog niet meteen kan toepassen. 

Zou dus eerder dit herschrijven weg van de oude memberManager en FamilyManager. 

In `frontend/shared/components/src/members/MemberStepView.vue` kan je zien hoe je een PlatformMember kan opslaan (= verzameling van data van een lid + alle wijzigingen eraan zitten in die class). Hoe je wijzigingen moet maken aan die PlatformMember kan je zien in één van de componenten in `frontend/shared/components/src/members/components/edit`